lib/stellar/client.rb

```diff
@@ -108,25 +109,29 @@ def create_account(options={})
       amount:   Stellar::Amount
     }) => Any
     def send_payment(options={})
-      from     = options[:from]
-      sequence = options[:sequence] || (account_info(from).sequence.to_i + 1)
+      from_account     = options[:from]
+      tx_source_account = options[:transaction_source] || from_account
+      op_source_account = from_account if tx_source_account.present?
```

Comment:
`tx_source_account` will always be `present`, and equal `options[:from]` or `options[:transaction_source]` right? It's important as it's used later in line 128.

Reply:
Yes, the tx source will be `:transaction_source` if present, then fallback to `:from`.